.github/workflows/release.yaml

```diff
@@ -0,0 +1,35 @@
+name: Release to charmhub
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    uses: ./.github/workflows/ci.yaml
+
+  publish:
+    name: Release to CharmHub
+    needs:
+      - build
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Check libs
+        uses: canonical/charming-actions/check-libraries@2.1.0
+        with:
+          credentials: "${{ secrets.CHARMHUB_TOKEN }}"
```

Comment:
Hi @taurus-forever and @deusebio,

Requesting to populate following secrets in this repo  (spark history server charm) to support publish to charmhub.

- secrets.CHARMHUB_TOKEN
- secrets.GITHUB_TOKEN

Reply:
GITHUB_TOKEN is auto-generated atuomatically on every run.

I can generate CHARMHUB_TOKEN after publishing this charm:
1) I do not see it: https://charmhub.io/spark-history-server nor https://charmhub.io/spark-history-server-k8s , have your registered name?
2) the name in metadata.yaml is `spark-history-server`, but it is a K8s operator. Should be fixed?

I can register charmhub name for you, but tell me the expected one.
P.S. https://charmhub.io/spark-k8s is on Dominik (Kubeflow).

Reply:
> P.S. https://charmhub.io/spark-k8s is on Dominik (Kubeflow)

We should not worry about this repo. It is a micro-service from kf we don't want to replicate in our spark stack

Reply:
We should still create the space in charmhub, AFAIK. @welpaolo was about to help @averma-canonical to do so. Probably this part should just go to a different PR, as it is another ticket 

The name of the space in charmhub I believe should just be `spark-history-server-k8s` to be consistent with other repo. Do you agree @taurus-forever ?

Reply:
Will work on release to charmhub in a separate PR as Enrico suggested.